client/my-sites/github-deployments/deployment-run-logs/deployment-run-logs.tsx

```diff
@@ -1,75 +1,103 @@
-import { translate } from 'i18n-calypso';
+import { useI18n } from '@wordpress/react-i18n';
 import { useMemo, useState } from 'react';
-import { LogEntry } from './use-code-deployment-run-log-query';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { GitHubLoadingPlaceholder } from '../components/loading-placeholder';
+import { LogEntry, useCodeDeploymentsRunLogDetailQuery } from './use-code-deployment-run-log-query';
+import { DeploymentRun } from './use-code-deployment-run-query';
 
 interface DeploymentRunLogsProps {
 	logEntries: LogEntry[];
+	run: DeploymentRun;
 }
 
-const DeploymentRunLog = ( { entry }: { entry: LogEntry } ) => {
+const DeploymentRunLog = ( { entry, run }: { entry: LogEntry; run: DeploymentRun } ) => {
+	const { __ } = useI18n();
 	const [ detailExpanded, setDetailExpanded ] = useState( false );
-
+	const siteId = useSelector( getSelectedSiteId );
+	const deployment = run.code_deployment!;
 	const openDetail = () => setDetailExpanded( ( v ) => ! v );
 
+	const commandIdentifier = entry.context?.command.command_identifier;
+	const hasDetail = !! commandIdentifier;
+
+	const { data: logDetail, isLoading } = useCodeDeploymentsRunLogDetailQuery(
+		siteId,
+		deployment.id,
+		run.id,
+		commandIdentifier ?? null,
+		{
+			enabled: detailExpanded && hasDetail,
+			refetchInterval: 5000,
```

Comment:
I don't think we need to refetch this endpoint. It's not like it's going to change.

Reply:
@jeroenpf is there any necessity to poll this endpoint? Just to confirm.

Reply:
No, it won't change after its been logged so we don't need to poll it.